frontend/language/src/nb.json

```diff
@@ -1768,6 +1771,7 @@
   "ux_editor.component_title.RadioButtons": "Radioknapper",
   "ux_editor.component_title.RepeatingGroup": "Repeterende gruppe",
   "ux_editor.component_title.Summary": "Oppsummering",
+  "ux_editor.component_title.Tabs": "Tabs",
```

Comment:
Er det et norsk ord vi kunne brukt her? 🤔  `Faner` kanskje?

Reply:
Nice catch! This text was just a test, and is not part of this PR 🤦‍♂️